cpp/pybind/t/geometry/trianglemesh.cpp

```diff
@@ -734,11 +734,14 @@ This function always uses the CPU device.
 
         o3d.visualization.draw([{'name': 'difference', 'geometry': ans}])
 )");
-
+    triangle_mesh.def("compute_adjacency_list", &TriangleMesh::ComputeAdjacencyList,
+    "Return Mesh Adjacency Matrix in CSR format using Triangle indices attribute.");
+    
```

Comment:
I am guessing that adjst() would be added to vertex attribute and adjv() to triangle attributes ? 

However the check HasVertexAttribute will fail because it checks the length of attribute is equal to the number of vertices, however as per CSR format the length of adjv is "num_vertices + 1"